### PR TITLE
[OYS-80]: Make BRANCH AMENITIES fieldset collapsible

### PR DESCRIPTION
**Pull request by AleksandrPrykhodkoDev**
Original issue:
https://github.com/ymcatwincities/openy/issues/1133
Branch nodes have fieldset with fields to enter amenities.
Would be great to make this fieldset collapsible and collapsed by default.

Steps for review

- [ ] check code changes
- [ ] visit page `node/add/branch`
- [ ] branch nodes have fieldset with fields to enter amenities
Снимок экрана 2020-02-21 в 11 57 05

### CODE REVIEW

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

File: modules/openy_features/openy_location/modules/openy_loc_branch/openy_loc_branch.install [hunk 1]
```diff
@@ -377,3 +377,15 @@ function openy_loc_branch_update_8017() {
...
 }
+
+/**
+ * Update feature configs.
+ */
+function openy_loc_branch_update_8018() {
+  $config_dir = drupal_get_path('module', 'openy_loc_branch') . '/config/install/';
+  $config_importer = \Drupal::service('openy_upgrade_tool.importer');
+  $config_importer->setDirectory($config_dir);
+  $config_importer->importConfigs([
```
Comment (AndreyMaximov) [inline modules/openy_features/openy_location/modules/openy_loc_branch/openy_loc_branch.install]: I guess updating the config partially might work better here. See the preview update hook as an example.

**Comment by AndreyMaximov** [comment]
ok to test

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2064/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2064/

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build2064/

**Review by podarok — DISMISSED** [review]
Please fix @AndreyMaximov review to get this merged
Thank you

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2066/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2066/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build2066/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build2066/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build2066/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build2066/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build2066/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2068/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2068/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build2068/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build2068/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build2068/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build2068/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build2068/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build2066/

**Pull request merged**